db/migrate/20160927071039_create_notification_blueprints.rb

```diff
@@ -0,0 +1,12 @@
+class CreateNotificationBlueprints < ActiveRecord::Migration
+  def change
+    create_table :notification_blueprints do |t|
+      t.string :group, index: true
+      t.string :level
```

Comment:
its not, its just a way for us to distinguish one notification from another for example:
```new_notification group: provisioning, name: 'finished with an error'```
vs
```new_notification :group: provisionining, message: <full message ...>```